.github/workflows/test-coverage-enhancement.yml

```
@@ -14,14 +14,14 @@
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Fetch all history to properly determine base
       - name: Set up Node.js
         uses: actions/setup-node@v2
```

actionlint (1.7.4)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)